selfdrive/boardd/boardd.cc

```diff
@@ -165,27 +176,116 @@ bool usb_connect() {
   }
 
   connected_once = true;
-  panda = tmp_panda.release();
+  if (tmp_panda->hw_type != cereal::PandaState::PandaType::UNO && tmp_panda->hw_type != cereal::PandaState::PandaType::DOS) {
```

Comment:
This would break EON gold or PC with black panda. Can we rewrite the check such that those keep working?

Reply:
I assume that secondary panda will be supported only by Comma 3, so I can check for DOS as primary and only on C3.